src/js/component.js

```diff
@@ -220,28 +220,35 @@ class Component {
    * @return {string}
    *         The localized string or if no localization exists the english string.
    */
-  localize(string) {
+  localize(string, tokens, defaultValue = string) {
```

Comment:
Why wouldn't we just make "{1} of {2}" the string we are localizing here? Seems weird to make the translation key and the value have different wording.